tests/models/hydrology/test_hydrology_model.py

```diff
@@ -390,17 +402,17 @@ def test_setup(
             coords={"cell_id": [0, 1, 2]},
         )
         exp_vertical_flow = DataArray(
-            [1.111498, 1.114365, 1.11434],
+            [0.672665, 0.67353, 0.673522],
             dims=["cell_id"],
             coords={"cell_id": [0, 1, 2]},
         )
         exp_soil_evap = DataArray(
-            [16.433136, 16.433136, 16.433136],
+            [347.118768, 346.741376, 347.120086],
```

Comment:
note: this is a bit higher than I would expect, but can be due to the inputs I chose. I'll investigate this further, I've added a TODO